CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 ### Fixed
 
+- `sandstone/ContextualPopupDecorator` to update popup position properly when the screen orientation change
```

Comment:
I think we need to sync this branch with the latest 2.9.x.develop and put this log accordingly.

Reply:
Yes, I readjusted the log location after synchronization